src/foam/u2/view/ChoiceView.js

```diff
@@ -238,7 +238,7 @@ foam.CLASS({
             .end();
         }
 
-        return self.E().add(self.text$);
+        return self.E().translate(self.text + ".name", self.text);
```

Comment:
What is 'name' at this point? Asking I expect to be translating the 'label'.

Reply:
According to docs, "this.text is the current label, this.label is the label for the whole view (eg. "Medal Color", not "Gold")."

RE: https://github.com/kgrgreer/foam3/pull/659/files#diff-eccdd95b711bd341a4fe353c6d21c0e1fdc2757568deae2515ae25c1b6d52449R32

Reply:
for translate(source, defaultValue) method call, the `source` is the choice view text + ".name" to be consistent with the Select tag, the `defaultValue` is the choice view text itself if it doesn't have translation.

Reply:
.name is compatible with the Select in RW mode, so is correct.